netci.groovy

```diff
@@ -844,14 +844,14 @@ combinedScenarios.each { scenario ->
                                     break
                                 case 'arm64':
                                     assert scenario == 'default'
-                                    buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} skiptestbuild /toolset_dir C:\\ats"
+                                    buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} /toolset_dir C:\\ats"
```

Comment:
The CI changes in the netci.groovy  are not reflected in the PR carying the change. So just checking in the change without above will not be of any use till the above goes in